draft-ietf-quic-multipath.md

```diff
@@ -351,31 +314,41 @@ does not discuss when a client decides to initiate a new path. We
 delegate such discussion in separate documents.
 
 To let the peer open a new path, an endpoint needs to provide its peer with connection IDs
-and associated Path Identifiers for the new path.
-
-To open a new path, an endpoint SHALL use different Connection IDs on different paths.
-Still, the receiver may observe the same Connection ID used on different
-4-tuples due to, e.g., NAT rebinding. In such case, the receiver reacts
+for at least one unused path identifier. Endpoints SHOULD use the MP_NEW_CONNECTION_ID
+frame to provide new connection IDs and, respectively, the MP_RETIRE_CONNECTION_ID frame to
+retire connection IDs after a successful handshake indicating multipath support by both endpoints.
```

Comment:
```suggestion
for at least one unused path identifier. Connection IDs for paths with Path IDs other than 0, can only
be announced and retired with the new MP_NEW_CONNECTION_ID and MP_RETIRE_CONNECTION_ID frames.
After a successful handshake indicating multipath support by both endpoints, endpoints SHOULD also use the MP_NEW_CONNECTION_ID and MP_RETIRE_CONNECTION_ID frames to provide new connection IDs or, respectively, retire them for the initial path with Path ID 0.
```

Reply:
True. However, this is meant to be an editorial PR. Given the SHOULD is already in the current version, maybe it cleaner to open and issue and then remove that eventually with a separate PR?

Reply:
Let's track this in #337 then.